bids2table/_b2t.py

```diff
@@ -24,6 +24,7 @@ def bids2table(
     workers: Optional[int] = None,
     worker_id: Optional[int] = None,
     return_table: bool = True,
+    exclude: Optional[List[str]] = None,
```

Comment:
Can you add this new arg to the docstring below? Also I would put the `exclude` arg just after `index_path`.